Hip 70 notes (#3834)
ROCm-Developer-Tools/HIP: docs/how-to/hip_runtime_api/error_handling.rst

.. note::

>     :cpp:func:`hipGetLastError` and ``cudaGetLastError`` returns the error
>     returned by any of the preceding HIP or CUDA APIs in the same host thread.
>     Before ROCm release 7.0, ``hipGetLastError`` did not match the ``cudaGetLastError`` behavior,
>     and instead returned the error code of the last HIP runtime API call even if it was ``hipSuccess``.

Author: ```suggestion
    :cpp:func:`hipGetLastError` returns the last actual HIP API error caught in the current thread
    during the application execution. Prior to ROCm 7.0, ``hipGetLastError`` might also return
    ``hipSuccess`` or ``hipErrorNotReady`` from the last HIP runtime API call, which are not errors.
```

.. note::

    :cpp:func:`hipGetLastError` and ``cudaGetLastError`` returns the error
    returned by any of the preceding HIP or CUDA APIs in the same host thread.

>     Before ROCm release 7.0, the `hipGetLastError` mismatched the bhaviour with

Reviewer: ```suggestion
    Before ROCm release 7.0, the `hipGetLastError` mismatched the behavior with
```

>     Before ROCm release 7.0, the `hipGetLastError` mismatched the bhaviour with
>     ``cudaGetLastError`` and returns the returned error code of the last HIP
>     runtime API call even if it's ``hipSuccess``.

Author: ```suggestion
    Before ROCm release 7.0, ``hipGetLastError`` did not match the ``cudaGetLastError`` behavior,
    and instead returned the error code of the last HIP runtime API call even if it was ``hipSuccess``.
```
